### DISCUSSION TITLE
get_user_followers

### DISCUSSION THREAD

**Discussion by AJKnuppe** [cjbarrie/academictwitteR#221] [Q&A]
Hi All,

I'm a relative novice at GitHub, so apologies in advance for any breaches in etiquette. 

I'm looking to scrape all the followers of a given account using the `get_user_followers` function listed in a previous version of the package vignette (accessible here: https://rdrr.io/cran/academictwitteR/man/get_user_followers.html).

I've installed the academictwitterR package from GitHub (using devtools) but am not seeing the `get_user_followers` in the current functions. Is this feature still in the development or test phase?

Thanks, in advance, for assistance.

Best,

Austin

**Comment by chainsawriot** [comment]
@AJKnuppe No, it is available even in the CRAN version.

Please uninstall `academictwitteR` and reinstall.

```r
remove.packages("academictwitteR")
install.packages("academictwitteR")
```

**Reply by AJKnuppe** [reply]
@chainsawriot thanks for the guidance, once I reinstalled from CRAN instead of GitHub the function appeared. As a follow up, is there an equivalent function to `get_retweeted_by` for capturing the usernames that reply to a given tweet?

**Reply by cjbarrie** [reply]
Hi Austin. This can be achieved by looking into conversation threads and associated functions. Please do consult the package documentation before asking questions like these--we have a lot of issues to respond to!
